test/e2e_node/e2e_remote.go

```diff
@@ -99,12 +99,8 @@ func CreateTestArchive() (string, error) {
 	if _, err := os.Stat(kubelet); err != nil {
 		return "", fmt.Errorf("failed to locate binary %s", kubelet)
 	}
-	apiserver := filepath.Join(buildOutputDir, "kube-apiserver")
-	if _, err := os.Stat(apiserver); err != nil {
-		return "", fmt.Errorf("failed to locate binary %s", apiserver)
-	}
 	ginkgo := filepath.Join(buildOutputDir, "ginkgo")
-	if _, err := os.Stat(apiserver); err != nil {
+	if _, err := os.Stat(ginkgo); err != nil {
 		return "", fmt.Errorf("failed to locate binary %s", ginkgo)
 	}
```

Comment:
Optional cleanup:

There's a lot of duplicated code in this method. How about just making a slice of required binaries, e.g.:
```go
requiredBins := []string{"kubelet", "e2e_node.test", "ginkgo"}
for _, bin := range requiredBins {
  // Check file exists
}
... // create archive dir
for _, bin := range requiredBins {
  // Copy to archive dir
}
```